apps/web/src/services/cost-explorer/budget/budget-detail/modules/budget-summary/BudgetSummaryTable.vue

```diff
@@ -56,7 +56,7 @@ type Providers = BudgetModel['provider_filter']['providers'];
 const budgetPageStore = useBudgetDetailPageStore();
 const budgetPageState = budgetPageStore.$state;
 
-const getAccumulatedBudgetUsageData = (budgetUsageData: BudgetUsageModel[], period: Period) => getStackedChartData(budgetUsageData, period, 'month');
+const getAccumulatedBudgetUsageData = (budgetUsageData: BudgetUsageModel[], period: Period) => getStackedChartData(budgetUsageData, GRANULARITY.MONTHLY, period);
```

Comment:
nitpick: I think the order of `getStackedChartData`'s arguments is wrong.

```suggestion
const getAccumulatedBudgetUsageData = (budgetUsageData: BudgetUsageModel[], period: Period) => getStackedChartData(budgetUsageData, period, GRANULARITY.MONTHLY);
```